Lib/bdb.py

```diff
@@ -306,7 +306,12 @@ def dispatch_line(self, frame):
         self.user_line(). Raise BdbQuit if self.quitting is set.
         Return self.trace_dispatch to continue tracing in this scope.
         """
-        if self.stop_here(frame) or self.break_here(frame):
+        # GH-136057
+        # For line events, we don't want to stop at the same line where
+        # we issue the previous next/step command.
```

Comment:
If we stop, it's not necessarily due to a next/step, right? Could it also be a breakpoint or until, etc?

So maybe the comment can say:

If we are stopping due to a next/step command, we don't want to stop on the same line on which
the command was issued.

Reply:
So I'm thinking. If we are currently at line 10, and the user does a `until 10`, should we stop at the current line? And if we stopped by a breakpoint, then we do `continue`, should we stop at the same line? Those are kind of undefined behaviors, unlike `step` and `next` which explicitly said that the debugger should stop at the "next" line.